services/backend/src/routes/connectivity.js

```diff
@@ -80,7 +80,8 @@ module.exports = (errorHandler, connectivityService) => {
    * @apiName DPCLayer
    * @apiVersion 1.0.0
    * @apiDescription
-   * Layers of the protected areas with higher dPC value in a given area
+   * Layers of the protected areas with higher dPC value in a given area. If paNumber is not
```

Comment:
Mark the param as optional in the documentation

Reply:
Done